content/posts/irods-development-update-november-2023.md

```diff
@@ -0,0 +1,24 @@
+Title: iRODS Development Update: November 2023
+Date: 2023-11-28 15:00
+Author: Kory Draughn
+Slug: irods-development-update-november-2023
+Status: published
+
+
+Hello Everyone,
+
+We're finally back from Supercomputing and it went very well. For those who attended, we hope you had as good a time as us. For those who did not attend, maybe we'll see you next year.
+
+Now for this month's update.
+
+With [iRODS 4.3.1 finally released](https://irods.org/2023/10/irods-4-3-1-is-released/), we're going to focus the beam on projects surrounding the server. That means addressing long standing issues in plugins and clients.
+
+First up is the iRODS S3 API. We've improved support for the MinIO client, added a configuration option which allows administrators to define the region, and added support for the `GetObjectLockConfiguration` API operation. We've also containerized the tests around the project. Now anyone can run the tests and build confidence using only `docker compose`. v0.1.0 was released earlier this month. Click [here](https://irods.org/2023/11/initial-release-of-the-irods-s3-api/) to read all about it.
```

Comment:
please just make 'v0.1.0 was released earlier this month' the link to the news, no need for the last sentence.